kie-user-bom-parent/pom.xml

```diff
@@ -74,6 +88,33 @@
       </snapshots>
     </repository>
   </repositories>
```

Comment:
Why do we have repositories in kie-user-bom-parent's pom ?

In the root pom (kie-parent), it makes sense to have repositories, to bootstrap the build.
But in kie-user-bom-parent I would guess that if the user didn't already have the right repo, he wouldn't find the bom in the first place?

Note: also consider the productized version of this: does that have the maven customer repo instead?